src/Core/Services/Implementations/OrganizationDomainService.cs

```diff
@@ -53,7 +53,7 @@ public async Task ValidateOrganizationsDomainAsync()
                 {
                     _logger.LogInformation(Constants.BypassFiltersEventId, "Successfully validated domain");
 
-                    //update entry on OrganizationDomain table 
+                    //update entry on OrganizationDomain table
```

Comment:
```suggestion
                    // Update entry on OrganizationDomain table
```
You're already removing whitespace at the end of these comments 🤷‍♂️